tcms/kiwi_auth/views.py

```diff
@@ -21,16 +21,16 @@
 
 User = get_user_model()  # pylint: disable=invalid-name
 
-
+# pylint: disable=R4511
```

Comment:
The disable directive goes on the line after the class name. Otherwise it means disable this checker from the current line until the end of the file.

Reply:
What does `R4511` mean  and how do I find its meaning ? Use human readable error codes, not numbers.